Flow.Launcher/SettingWindow.xaml

```diff
@@ -171,6 +171,7 @@
                     IsPaneToggleButtonVisible="False"
                     IsSettingsVisible="False"
                     IsTabStop="False"
+                    Loaded="NavView_Loaded"
```

Comment:
shouldn't previously `Loaded` is `OnLoaded`?

Reply:
Sorry, what does that mean?

Reply:
@taooceros could you also explain what you mean please.

Reply:
Sorry I misunderstood. Please ignore